tests/caasp/stack_controller.pm

```diff
@@ -80,13 +80,20 @@ sub velum_bootstrap {
     # Wait until warning messages disappears
     wait_still_screen;
 
-    # Click next button to 'Confirm bootstrap' page
-    send_key_until_needlematch 'velum-next', 'pgdn', 2, 5;
-    assert_and_click 'velum-next';
+    if (check_var('DISTRI', 'caasp') && get_var('VERSION', '2.0')) {
```

Comment:
This change will be incompatible with Kubic

Reply:
Nice catch! I have no idea what's the correct settings for Kubic. Could you please post here in the comment section what this conditional for Kubic would look like? Then, I will `commit --amend` this conditional including Kubic ;)

The next question is, what Kubic is based on? 1.0 or 2.0 ?

Reply:
Maybe relocating the version check, by adjusting `2.0` in the `else` and checking only for `1.0`. My guess that I would like to pro-actively make sure that version 4 (given that it will be compatible with 2.0) it will not be broken or need an extra `if`. Would that be better for Kubic or it's completely not related?